src/claude-history-dashboard/src/styles.css

```
@@ -1,380 +1,6 @@
 @import 'tailwindcss';
+@source "../../../utils/ui";
 @import 'tw-animate-css';
```

Stylelint (17.4.0)

[error] Unexpected invalid position `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)